src/main/plugins/org.dita.html5/xsl/rel-links.xsl

```diff
@@ -578,6 +579,12 @@ Each child is indented, the linktext is bold, and the shortdesc appears in norma
   </xsl:template>
 
   <!--linklists-->
+  <xsl:template match="*[contains(@class, ' topic/linklist ')]/@xml:lang" priority="100">
```

Comment:
This section ensures that if the `@xml:lang` value on the link list matches an already-processed ancestor language (such as the language on a root `<topic>`, we don't need to reset `lang` to the same value in the generated HTML5. It also skips doing this if there is no ancestor language and this link list uses the default language. Basically, just a way to keep extra noise out of the HTML.